tests/testthat/test-all.R

```diff
@@ -255,11 +255,11 @@ test_that("read Socrata JSON with missing fields (issue 19 - bind within page)",
 
 test_that("read Socrata JSON with missing fields (issue 19 - binding pages together)", {
   ## Define and test issue 19
-  expect_error(df <- read.socrata(paste0("https://data.smgov.net/resource/ia9m-wspt.json?",
-                                         "$where=incident_date>'2010-12-15'%20AND%20incident_date<'2011-01-15'"))
-               , NA, info = "https://github.com/Chicago/RSocrata/issues/19")
-  expect_equal(7927, nrow(df), label="rows", info = "https://github.com/Chicago/RSocrata/issues/19")
-  expect_equal(18, ncol(df), label="columns", info = "https://github.com/Chicago/RSocrata/issues/19")
+  df <- read.socrata(paste0("https://data.smgov.net/resource/ia9m-wspt.json?",
+                            "$where=incident_date>='2011-01-01'%20AND%20incident_date<'2011-01-15'"))
```

Comment:
@geneorama - wondering why the dates of the query was changed?

Reply:
We needed all the values in the column for the JSON column to be incomplete to test the fill.  For some reason the range of NA values appears to have changed and now some of the NA values are no longer NA. 

This was very confusing and I spent a lot of time on it, started to write up extensive comments, but then I wasn't sure of our starting point, so there was nothing to clearly document. Why did we pick that exact date range in the first place? I'm not sure.